samples/04_gis_analysts_data_scientists/covid19_part1_mapping_the_pandemic.ipynb

```diff
@@ -53,7 +53,7 @@
   },
```

Comment:
<p>One more paragraph needs to update for the new functions and renderer, and then good to go :) Thanks John for updating all the narratives!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1984/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Reply:
<p>done!</p>